packages/blade/tsconfig-generate-types.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": ["**/**.test.ts", "**/**.test.tsx", "__tests__"],
+  "include": ["src", "src/tokens/theme/theme.d.ts"],
```

Comment:
Why do we need to include `"src/tokens/theme/theme.d.ts"` specifically? Does it not pick things from `src` or `src/**`?